iam/api/AccessTest/AccessTest.cs

```diff
@@ -124,5 +123,11 @@ public void TestAccess()
                 service.Projects.Roles.Delete(role1.Name).Execute();
             }
         }
+
+        [Fact]
+        public void TestPermissions()
+        {
+            var permissions = AccessManager.TestPermissions(_project);

```

Comment:
You don't have any asserts here. What is the purpose of the test?

Reply:
I'm expecting TestPermissions to throw an error if it fails (following the pattern of [QuickStartTest.cs](https://github.com/melaniedejong/dotnet-docs-samples/blob/master/iam/api/QuickStartTest/QuickStartTest.cs)). I'm not entirely sure what to assert--is there a standard for C# tests?

Reply:
Nothing to assert.  Your test is fine.